runelite-client/src/test/java/net/runelite/client/plugins/cannon/CannonPluginTest.java

```diff
@@ -168,23 +129,23 @@ public void testThresholdNotificationShouldNotifyOnce()
 	public void testThresholdNotificationsShouldNotNotify()
 	{
 		when(config.showCannonNotifications()).thenReturn(true);
-		when(config.lowWarningThreshold()).thenReturn(0);
 
-		cannonAmmoChanged.setValue(30);
+		cannonAmmoChanged.setValue(22);
 		plugin.onVarbitChanged(cannonAmmoChanged);
-		cannonAmmoChanged.setValue(10);
+		cannonAmmoChanged.setValue(0);
 		plugin.onVarbitChanged(cannonAmmoChanged);
 
-		verify(notifier, never()).notify("Your cannon has 10 cannon balls remaining!");
+		verify(notifier, never()).notify("Your cannon is out of ammo!");
```

Comment:
This test change causes the test name to be completely inaccurate. Also, this ends up being exactly the same as `testCannonOutOfAmmo()` below with your changes.